EXAMPLES.md

```diff
@@ -467,3 +469,56 @@ export const getServerSideProps = (ctx) => {
   return withPageAuthRequired({ returnTo })(ctx);
 };
 ```
+
+## Use a custom session store
+
+You need to create your own instance of the SDK in code, so you can pass an instance of your session store to the SDK's configuration.
+
+```typescript
+// lib/auth0.ts
+import { SessionStore, SessionStorePayload, initAuth0 } from '@auth0/nextjs-auth0';
+
+class Store implements SessionStore {
+  private store: KeyValueStoreLikeRedis<SessionStorePayload>;
+  constructor() {
+    // If you set the expiry accross the whole store use the session config,
+    // e.g. `min(config.session.rollingDuration, config.session.absoluteDuration)`
+    // the default is 24hrs
+    this.store = new KeyValueStoreLikeRedis();
+  }
+  async get(id) {
+    const val = await this.store.get(id);
+    return val;
+  }
+  async set(id, val) {
+    // To set the expiry per item, use `val.header.exp` (in secs)
+    const expiryMs = val.header.exp * 1000;
```

Comment:
Where is `expiryMs` used?

Reply:
Fair, point - it depends on the store, will add an example for redis

```suggestion
    const expiryMs = val.header.exp * 1000;
    // Example for Redis: redis.set(id, val, { pxat: expiryMs });
```